utils/vcsdetails.go

```diff
@@ -13,10 +13,10 @@ import (
 type (
 	VcsCache struct {
 		// Key - Path to the .git directory.
-		// Value - Reference to a struct, storing the URL and revision.
+		// Number - Reference to a struct, storing the URL and revision.
```

Comment:
Let'e revert this change, because "value", indicates the map's value. "Number" seems wrong.